wiki/Community/Project_Loved/en.md

```diff
@@ -31,15 +31,13 @@ There are some very minimal criteria that need to be met for beatmaps to be nomi
 - At least one difficulty can be passed
 - Every nominated difficulty has a [drain time](/wiki/Beatmap/Drain_time) of at least 20 seconds
 
-Although many unranked beatmaps meet these criteria, few are voted into the Loved category due to the selective nature of the captains' picks, and the high community approval requirement by voting.
-
 ### Updates prior to Loved
 
 In some cases, beatmaps may be nominated despite breaking some of the criteria, as long as the mapper can update it to abide by the criteria prior to Loved. For example, a beatmap with an inappropriate background may be nominated as long as the background is later removed or replaced. An active contributor to Project Loved will usually post on the beatmap's [discussion](/wiki/Beatmap_discussion) if any changes are required.
 
 Additionally, contributors to Project Loved often make technical suggestions, such as improvements to audio quality, [timing](/wiki/Beatmapping/Timing) accuracy, or [metadata standardisation as recommended by the ranking criteria](/wiki/Ranking_criteria/Metadata). The mapper is strongly encouraged to apply these suggestions, but they are not required to.
 
-### osu!-specific criteria
+### osu!standard-specific criteria
```

Comment:
osu!standard is no longer officially-supported terminology, you just call it `osu!`.

Reply:
Shall do